<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>684</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>connect_MR9270S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_disconnect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>disconnect_MR9270S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_read">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>read_MR9270S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_write">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>write_MR9270S(start)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XMT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ohm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RTD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TC</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_readout">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_writein">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_outputtype">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_linkstate1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_linkstate2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_write_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>write_MR9270S(stop)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_outputstate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_comm">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_outputtype_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号选择：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
